src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
Great job on following the naming conventions (using plural forms for collection of objects)